src/groupByKey.ts

```diff
@@ -2,6 +2,16 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const obj: GroupsMap<T> = {};
+
+  items.forEach((item) => {
+    if (obj[`${item[key]}`]) {
```

Comment:
why not hasOwnProperty?

Reply:
When i using hasOwnProperty linter throws an error